### PR TITLE
Fix couchdb-cluster-admin dependency

### PR DESCRIPTION
**Pull request by pr33thi**
Couchdb-cluster-admin upgraded and one of the package names changed. Upgraded the version number in `setup.py`, and changed the package name

### CODE REVIEW

File: src/commcare_cloud/commands/migrations/couchdb.py [hunk 1]
```diff
@@ -233,11 +233,10 @@ def prepare_to_sync_files(migration, ansible_context):
     return rsync_files_by_host
 
 
-def generate_rsync_lists(migration, validate_suffixes=True):
+def generate_rsync_lists(migration):
     full_plan = {plan.db_name: plan for plan in migration.shard_plan}
-    important_files_by_node = get_important_files(
-        migration.source_couch_config, full_plan, validate_suffixes=validate_suffixes
```
Comment (emord) [inline src/commcare_cloud/commands/migrations/couchdb.py]: Does this mean that couchdb_cluster_admin automatically validates suffices now?

**Comment by emord** [comment]
looks like a non trivial failure.

@snopoke guessing you'll know the most about these changes
```
======================================================================
ERROR: tests.test_migration.test_migration_plan('reshard',)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose_parameterized/parameterized.py", line 280, in <lambda>
    nose_func = wraps(func)(lambda *args: func(*args))
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/home/travis/build/dimagi/commcare-cloud/tests/test_migration.py", line 35, in test_migration_plan
    files_by_host = generate_rsync_lists(migration)
  File "/home/travis/build/dimagi/commcare-cloud/src/commcare_cloud/commands/migrations/couchdb.py", line 239, in generate_rsync_lists
    migration.source_couch_config, full_plan)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/couchdb_cluster_admin/file_plan.py", line 134, in get_node_files
    return figure_out_what_you_can_and_cannot_delete(plan, _get_shard_suffixes(config, plan))
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/couchdb_cluster_admin/file_plan.py", line 88, in _get_shard_suffixes
    cluster_allocation_doc = get_shard_allocation(config, db_name)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/couchdb_cluster_admin/utils.py", line 78, in get_shard_allocation
    shard_allocation_doc = ShardAllocationDoc.wrap(do_node_local_request(node_details, '_dbs/{}'.format(db_name)))
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/couchdb_cluster_admin/utils.py", line 22, in do_node_local_request
    return _do_request(node_details, path, node_details.node_local_port, method=method, params=params, json=json)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/couchdb_cluster_admin/utils.py", line 38, in _do_request
    proxies=proxies
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/requests/api.py", line 58, in request
    return session.request(method=method, url=url, **kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/requests/sessions.py", line 508, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/requests/sessions.py", line 618, in send
    r = adapter.send(request, **kwargs)
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/requests/adapters.py", line 508, in send
    raise ConnectionError(e, request=request)
ConnectionError: HTTPConnectionPool(host='10.247.164.12', port=15986): Max retries exceeded with url: /_dbs/commcarehq (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f12cc203e10>: Failed to establish a new connection: [Errno 110] Connection timed out',))
-------------------- >> begin captured logging << --------------------
urllib3.connectionpool: DEBUG: Starting new HTTP connection (1): 10.247.164.12
--------------------- >> end captured logging << ---------------------
```

**Comment by pr33thi** [comment]
I messaged @snopoke but I'll put it here so we can all see it:

I spoke to @dannyroberts , and he suggested that `validate_suffixes` would only be set to false for tests, is that accurate?

**Comment by snopoke** [comment]
Apologies for the mess up, this PR was intended to go out with the upgrade. I think the simplest solution is to just pin the version at 0.2.1: https://github.com/dimagi/commcare-cloud/pull/1799

**Pull request closed without merging**